src/Page/Food/Builder.elm

```diff
@@ -760,6 +766,46 @@ packagingListView db selectedImpact recipe results =
     ]
 
 
+distributionView : Impact.Definition -> Recipe -> Recipe.Results -> List (Html Msg)
+distributionView selectedImpact recipe results =
+    [ div [ class "card-header d-flex align-items-center justify-content-between" ]
+        [ h5 [ class "mb-0" ] [ text "Stockage et distribution" ]
+        , results.recipe.conservation
+            |> Format.formatFoodSelectedImpact selectedImpact
+        ]
+    , div []
+        [ ul [ class "list-group list-group-flush border-top-0 border-bottom-0" ]
+            [ li [ class "IngredientFormWrapper" ]
+                [ select
+                    [ class "form-select form-select-sm"
+                    , onInput UpdateConservation
+                    ]
+                    (Retail.all
+                        |> List.map
+                            (\c ->
```

Comment:
One letter variables aren't very readable and don't ease maintenance, I'd stay away from them when possible at all (eg here, you could use `conservation` instead of `c`)

Reply:
Sure ! One letter variables are relevant only in fully generic code that work with any type ("forall" kinds)